wiki/Usage_permissions/en.md

```diff
@@ -0,0 +1,427 @@
+| Yunosuke                         ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/244)  |
+| Yuyoyuppe / DJ'TEKINA//SOMETHING ![](FA) | 🟠 Allowed, with exceptions         | Tracks from or themed around Touhou should not be uploaded. | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/132)  |
+| YUZUKINGDOM                      ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/155)  |
+| Zekk                             ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/76)   |
+| Zenpaku                          ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/316)  |
+| Zomboy                           ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/224)  |
+| ZxNX                             ![](FA) | 🟢 Allowed                          |                                                             | Officially licensed songs can be found at their [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/288)  |
+
+## Visual
+
+Visual works refer to elements like beatmap backgrounds and are commonly retrieved from sites like Pixiv.
+
+In many cases, a visual artist will make it immediately apparent if they permit the use of their work elsewhere.  All visual works should be properly attributed and provide an appropriate link-back to their source. If a direct link cannot be used for whatever reason, another link that points to the visual artist's account on their most active platform should be used instead.
+
+If it is unclear whether a visual artist allows their work to be used on other platforms, we recommend not making use of any until appropriate usage conditions can be confirmed.
+
+## Notes
+
+[^monstercat-gold]: This artist is licensed in cooperation with [Monstercat](https://osu.ppy.sh/beatmaps/artists/255), and users may be able to obtain external media usage rights via a [Monstercat Gold](https://www.monstercat.com/gold) subscription. Please consult their site for specific details on this license.
```

Comment:
is this supposed to be a limitation or just a note for your convenience on where to get "external media usage rights"? it's under the "Exceptions" column of the table but it doesn't really seem like one.

Reply:
The column should probably be renamed to "Notes".